src/main/java/uk/ac/ebi/subs/PropertiesManager.java

```diff
@@ -16,7 +16,9 @@ private PropertiesManager() {
         try (FileInputStream fileInputStream = new FileInputStream(path)) {
             properties.load(fileInputStream);
         } catch (FileNotFoundException e) {
-            throw new RuntimeException("ERROR: No application.properties file found.", e);
+            if(System.getProperty("aapUsername") == null || System.getProperty("aapPassword") == null) {
```

Comment:
I don't understand why is this check here. What is it to do with a `FileNotFoundException`?

Reply:
If there's no file and no AAP credentials provided on the command line, then the tests can't run, but we can still run them if there's no properties file and the credentials have been given in the command line.

Reply:
OK